### PR TITLE
ISPN-10054 Removed the tree module

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-10054

### CODE REVIEW

File: documentation/src/main/asciidoc/upgrading/upgrading-content.adoc [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 The previously deprecated DistributedExecutor is now removed. References should be updated to use ClusterExecutor.
 
+=== Removed the Tree module
+
+TreeCache has been unsupported for a long time and was only intended as a quick stopgap for JBossCache users. The module has now been removed
+completely.
```
Comment (danberindei) [inline documentation/src/main/asciidoc/upgrading/upgrading-content.adoc]: Nitpicking: as a rule, we should split lines to on `.`, although in this case there won't be any updates so it won't make any diffs nicer.

**Comment by anistor** [comment]
+1

**Comment by danberindei** [comment]
No reason to delay this any longer, integrated!

**Comment by anistor** [comment]
RIP tree. Earth to earth, ashes to ashes, sawdust to sawdust.

**Pull request merged**